File: app/src/main/java/com/orionedwards/mtglifecounter/RandomGen.java
package com.orionedwards.mtglifecounter;

import java.util.Random;

public class RandomGen {
    static final Random sRNG = new Random(12);

    public static int next(int exclusiveUpperBound) {
        return sRNG.nextInt(exclusiveUpperBound);
    }
}

File: app/src/main/java/com/orionedwards/mtglifecounter/RandomGen.kt
package com.orionedwards.mtglifecounter

import java.util.Random

object RandomGen {
    private val sRNG = Random(12)

    @JvmStatic
    fun next(exclusiveUpperBound: Int): Int {
        return sRNG.nextInt(exclusiveUpperBound)
    }
}
